pkg/collector/corechecks/embed/apm.go

```diff
@@ -21,6 +21,7 @@ import (
 	"github.com/DataDog/datadog-agent/pkg/collector/check"
 	core "github.com/DataDog/datadog-agent/pkg/collector/corechecks"
 	"github.com/DataDog/datadog-agent/pkg/config"
+	"github.com/DataDog/datadog-agent/pkg/telemetry"
```

Comment:
I wonder if this code is still used since the trace agent lifecycle is managed by systemd

Reply:
That's a good question, I have to look into that.

Reply:
This code is still in use on macOS.